Examples/SnapinsSDKExample/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.1'
+        classpath 'com.android.tools.build:gradle:3.3.2'
```

Comment:
Should probably create a task to update the build tools and kotlin version to the very latest (`3.4.0` and `1.3.30`).